### DISCUSSION TITLE
Game1 Genre Poll

### DISCUSSION THREAD

**Discussion by KaizNike** [Godot-Community-Games-Org/game1#5] [Polls]
I'm not officially added to this org yet, but as I wait, I got a few likes on my post of a call for a genre poll to decide, so let's start!

I'm limited to 8 options out of 100s of genres, so feel free to add your own and any upvotes for that can be compared against my selection.

**Comment by KaizNike** [comment]
I'm going to vote Visual Novel, and my concept would be that we can create a trunk that we all oversee and then branches of that trunk as different choice lines that individuals are responsible for and give their own distinct storyline to. There is a good deal of variety among visual novels in how they deal with branching content, we could take a few notes from the Fallen London/Sunless Sea team.

Otherwise most of my knowledge of open source game successes is in Roguelikes. (Cataclysm:DDA and Dungeon Crawl: Stone Soup)

**Comment by ryankmims** [comment]
I voted platformer because it has a smaller scope, and once the game objects are finalized people can build levels by just dragging and dropping, and then the community can then vote on which levels should be added to the game.

But we could also do that with puzzle games, and rougelikes.

**Comment by Yonodactyl** [comment]
I voted roguelike because I've never worked on one and I'm hoping to learn while having fun with this all! I think scope creep is a very real thing, so picking something relatively simple is going to be what we want here for sure, so I'm open to a lot of genres.

The visual novel trunk concept was pretty interesting though. I like the idea that work can be siloed and then we come together throughout points in time to merge the stories together.

**Comment by GodotCommunityGames** [comment]
Great idea for a poll. Also, as we set up this community and start discussing things, we might decide to start up several different games at the start, instead of just one big community game, based on what people are interested in. But yeah this poll is great for getting a first look at what genres people are gonna most be interested in and getting that game ideas discussion going!

**Reply by Morzisse** [reply]
it has been mentioned in the reddit post once but i dont know if it was mentioned anywhere else, but if the idea of doing multiple games comes into fruition how about a ufo 50 like approach to sort of connect the games into 1 hub?

**Comment by BlodyxCZ** [comment]
I voted roguelike because I think it will be a great starting project for it's simplicity over some of the others. I also think it will be the most fun to work at as a community, since you can basically add anything into a roguelike and it will work.

**Comment by AnonymousAggregator** [comment]
i vote roguelike for small scope. visual novel for second, as we could have wide community involvement potential.

**Comment by bubba169** [comment]
I'm thinking a roguelike could be good fun and easy to split into chunks. I'm picturing a wave based shooter/hack and slash with differently designed areas that you progress through randomly. Each area would have its own set of enemies. Completing a section would give you currency to buy weapon upgrades or single use boosts to start the next run with.

**Comment by coderKillo** [comment]
I also would go for roguelike. They are easy to manage from architecture and scope. Ive a few create a solid base many can create lots of content.
Besides that have developed 2 rogueLITE in the past and would be intrested in tapping more into the RPG part of this. Also a turn based game is still on my buckted list.

**Comment by fishinmycola** [comment]
I chose a platformer because it gives us a lot of flexibility, and it’s perfect for a collaborative project. The cool thing about it is how we can break down the work based on each person's strengths. For instance, someone with a great eye for design can focus on making the level look amazing, while a programmer can bring in cool, unique mechanics specific to that level.

It’s also a great way to keep things manageable. Platformers allow us to work modularly—each level can feel different but still be part of the same game. That way, everyone contributes something unique without it getting too complicated.

On the other hand, making something like a roguelike could easily become overwhelming. The amount of different mechanics and systems could confuse players if we don’t get the balance just right. Plus, roguelikes tend to need more cohesion to feel satisfying, which might be tough to pull off in our first group project.

**Comment by ghost** [comment]
I voted for Roguelike because as others have said; it's a nice small scope and I'd be interested in seeing what a lot of the systems boil down to in gdscript/C#.

coderKillo mentioned turned based games and I love the original Rogue's fast turn based gameplay, I think it would be cool to try this gameplay beat but maybe combine it with a less likely setting ? That could be interesting.

**Comment by DaRaKu2002** [comment]
I voted plattformer but im also very interested in rogue likes. I dont know if id want it to be top down tho. A side scroller rogue like would be interesting to make. Ive seen others mention turn based and im not too familiar with the Genre but definitly curious.

**Comment by SpearsDracona** [comment]
I picked strategy because it doesn't seem to be done as much as the others and I personally love strategy games. I love the idea of a community written visual novel, but I wonder if there's enough people here who are into the art and writing side of things to flesh it out. I think as this community grows it might be a fun idea for a game to do further down the line.

**Comment by TheBardWhoCodes** [comment]
I voted Online Multiplayer because that's the majority of what I play but I know that's a larger lift. Something like an online Card battler with simple mechanics or a barebones 2d shooter might not be too bad though.
